packages/aws-cdk/test/api/hotswap/hotswap-test-setup.ts

```diff
@@ -1,12 +1,12 @@
 import * as cxapi from '@aws-cdk/cx-api';
-import { CloudFormation } from 'aws-sdk';
+import { CloudFormationStackArtifact } from '@aws-cdk/cx-api';
```

Comment:
We have `import * as cxapi from '@aws-cdk/cx-api';` above, let's not import this class directly here.